winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -76,12 +76,24 @@ namespace Microsoft.Graphics.Canvas.Text
     [version(VERSION)]
     typedef struct CanvasGlyph
     {
-        int Index;
+        UINT16 Index;
```

Comment:
Making this a signed-int type was a deliberate decision; it fits better with WinRT patterns and precedents, even though it may not exactly match what DirectWrite returns. Besidea that, is there some reason to make it an unsigned short??